places/vicuk-london.md

```diff
@@ -0,0 +1,9 @@
+## London
+
+### Fun Facts
```

Comment:
Can you tell me a fun fact?  I've never been to London.  Hope I commented correctly. :)